<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Bhakti" failures="1" tests="4" name="C:/Users/BHAKTI/Selenium Automation Projects/FirstSeleniumWebdriverScript" time="8.38" errors="0" timestamp="2024-10-01T22:41:39 GMT-04:00">
  <testcase classname="TestNG.SwagLabsTest" name="launch" time="2.551"/>
  <testcase classname="TestNG.SwagLabsTest" name="loginTest" time="2.195"/>
  <testcase classname="TestNG.SwagLabsTest" name="currentUrlTest" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/inventory.htmll] but found [https://www.saucedemo.com/inventory.html]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.htmll] but found [https://www.saucedemo.com/inventory.html]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at TestNG.SwagLabsTest.currentUrlTest(SwagLabsTest.java:53)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
]]>
    </failure>
  </testcase> <!-- currentUrlTest -->
  <testcase classname="TestNG.SwagLabsTest" name="LogoutTest" time="2.105"/>
  <testcase classname="TestNG.SwagLabsTest" name="titleTest" time="0.0">
    <ignored/>
  </testcase> <!-- titleTest -->
</testsuite> <!-- C:/Users/BHAKTI/Selenium Automation Projects/FirstSeleniumWebdriverScript -->
